src/main/pages/che-7/installation-guide/proc_configuring-che-persistent-volumes-strategy.adoc

```diff
@@ -1,27 +1,17 @@
 [id="configuring-{prod-id-short}-persistent-volumes-strategy_{context}"]
```

Comment:
Same as the above comment. Should we get rid of it or is it a placeholder for some planned content?

Reply:
I am reworking this content. Will put it in Che `quick-starts` when ready.

Reply:
Oh, that's a different doc with a similar name..but anyway, it has its place in Installation Guide. Here is a plan for it:
https://github.com/eclipse/che/issues/14362